src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -476,13 +471,10 @@ internal void LoadLibraryMigrations(string library)
         {
             string fullLibraryName = library;
 
-            if (!pathManager.ResolveLibraryPath(ref fullLibraryName))
-                return;
-
             string migrationsXMLFile = Path.Combine(Path.GetDirectoryName(fullLibraryName),
                 Path.GetFileNameWithoutExtension(fullLibraryName) + ".Migrations.xml");
 
-            if (!File.Exists(migrationsXMLFile))
+            if (!pathManager.ResolveDocumentPath(ref migrationsXMLFile))
```

Comment:
Built in nodes also don't have assembly, that they came from. That's why `ResolveLibraryPath` won't give any result.